frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -318,16 +408,34 @@ export default {
         applicationModel,
       )
     },
-    getModel(type) {
-      return this.models?.find((el) => el.supplementaryType === type)
+    getModel(type, currentTerm) {
+      return this.models?.find((el) => el.supplementaryType === type && el.renewalTerm == currentTerm)
```

Comment:
I think this can use strict equality (===).

Reply:
done